src/main/resources/openapi.json

```diff
@@ -1,286 +1,1279 @@
+                    "content": {
+                        "application/json": {
+                            "schema": {
+                                "$ref": "#/components/schemas/Consumer"
+                            }
+                        }
+                    },
+                    "required": true
+                },
+                "responses": {
+                    "200": {
+                        "description": "Consumer created successfully.",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "$ref": "#/components/schemas/CreatedConsumer"
+                                },
+                                "examples": {
+                                    "response": {
+                                        "value": {
+                                            "instance_id": "consumer1",
+                                            "base_uri": "http://localhost:8080/consumers/my-group/instances/consumer1"
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    },
+                    "409": {
+                        "description": "A consumer instance with the specified name already exists in the Kafka Bridge.",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "$ref": "#/components/schemas/Error"
+                                },
+                                "examples": {
+                                    "response": {
+                                        "value": {
+                                            "error_code": 409,
+                                            "message": "A consumer instance with the specified name already exists in the Kafka Bridge."
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    },
+                    "422": {
+                        "description": "One or more consumer configuration options have invalid values.",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "$ref": "#/components/schemas/Error"
+                                },
+                                "examples": {
+                                    "response": {
+                                        "value": {
+                                            "error_code": 422,
+                                            "message": "One or more consumer configuration options have invalid values."
```

Comment:
Does the message really not include the names of the problematic option(s)? How do we expect people to debug their code?

Reply:
It's just an example, we will have different messages in order to give more details to the user.